tools/loader/ImageLoader.cpp

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Loader.h"
+
+#include "glow/Base/Image.h"
+#include "glow/Graph/Nodes.h"
+#include "glow/Importer/Caffe2.h"
+#include "glow/Importer/ONNX.h"
+
+#include "llvm/ADT/StringSwitch.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/raw_ostream.h"
+
+using namespace glow;
+
+enum class ImageNormalizationMode {
+  Undefined, // Value range is undefined.
```

Comment:
nit: Undefined is not actually needed. We could remove that from enum and make the command line argument to be required instead of optional.